ui/app/js/controllers/release_delete_controller.js

```diff
@@ -17,7 +17,7 @@ function ($scope, $modalInstance, CSRF, Releases, release, releases) {
       })
       .error(function(response) {
          if (typeof response === 'object') {
-          message = '';
+          message = response.exception;
```

Comment:
As discussed on IRC with tieu, the exception message in response is being assigned to the variable to be displayed on UI. Previously the message was set as blank, hence the UI wasn't displaying the error message.